media/css/exp/firefox/accounts-2019.scss

```diff
@@ -2,7 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../../protocol/css/includes/lib';
+@import '../../~@mozilla-protocol/core/protocol/css/includes/lib';
```

Comment:
Curious why this needs to be a relative path still?

If we're delving into `node_modules` for the source, could things like this just be `@import '~@mozilla-protocol/core/protocol/css/includes/lib';`?